### DISCUSSION TITLE
I'd like to offer my help on this project (SRS Doc, Bug, Feature)

### DISCUSSION THREAD

**Discussion by Nikitas-io** [DIYgod/APlayer#679] [General]
Hello there @DIYGod ! Recently I have been given an assignment by my University to find an Open Source project and try to offer my services in one of the 3 following ways:
1. Write a 30 page [SRS Document](https://winatalent.com/blog/2020/05/srs-document-the-what-the-why-and-the-how/) for that project 
2. Report a new issue and try to fix it.
3. Expand the project's functionality by creating a new feature.

- I have found this project (APlayer) and DPlayer extremely interesting because I'd like to incorporate them in my own projects in the future. So I decided to ask you if you'd like me to **create the [Software Requirements Specification (SRS Document)](https://www.perforce.com/blog/alm/how-write-software-requirements-specification-srs-document) for APlayer** or Dplayer (both are interesting but I'd prefer to write it for APlayer). Of course, I will be using the [IEEE standard](https://web.cs.dal.ca/~hawkey/3130/srs_template-ieee.doc) to create the SRS document.

- I have also [found an issue for this repo](https://github.com/DIYgod/APlayer/issues/677) and if possible, I'd like you to assign it to me.

- Alternatively, I could try to [add the feature that I proposed in this issue](https://github.com/DIYgod/APlayer/issues/678).

It's necessary for my assignment that I interact with the project's community so please respond to this message after you read it as soon as possible because **I do have a deadline** 🙏🏻

**Comment by DIYgod** [comment]
Sorry, I may not have a lot of time to work on open source projects in the near future, feel free to submit pr if you are interested, but I can't guarantee the time, minor changes would be quicker.

If you are pressed for time, I suggest you look for other projects that are more up-to-date.

**Reply by Nikitas-io** [reply]
Thank you for the answer
I have fixed issue #677 and created the #681 pull request for it. 
Kindly review the changes please :)
